sycl/include/CL/sycl/detail/pi.h

```diff
@@ -385,11 +385,12 @@ pi_result piContextRelease(pi_context context);
 //
 // Queue
 //
-pi_queue piQueueCreate( // TODO: change interface to return error code instead
-  pi_context              context,
-  pi_device               device,
-  const cl_queue_properties *    properties, // TODO: untie from OpenCL
-  pi_result *             result);
+pi_result piQueueCreate(
```

Comment:
thanks for fixing the TODO!
could you also add pi_queue_properties enum and populate it with what SYCL is using?

Reply:
Sure, I can give it a try, just to clarify do you mean: cl_command_queue_properties rather than cl_queue_properties?

The former is an enumerator the latter is an array of a few different properties cl_uint/cl_command_queue_properties and khr extensions.

Reply:
I mean  replace cl_command_queue_propertie with [new] pi_queue_properties enum

Reply:
thanks for the clarification! Made the change, hopefully I wasn't dumb and misinterpreted you.